.github/workflows/workflow.yml

```diff
@@ -43,129 +89,85 @@ jobs:
         files: tests/bake.hcl
         push: true
 
+
   build:
-    needs: lint
+    needs: tests
```

Comment:
It looks a bit confusing that `build` goes after `tests`. Probably `release` would be a better job name.

Reply:
Done